.github/workflows/docker.yml

```diff
@@ -1,12 +1,6 @@
 name: Build and upload Docker image
 
-on:
-  release:
-    types: [published]
-  push:
-    branches: [dev]
-    tags:
-      - "*.*.*"
+on: push
```

Comment:
question if we should have docker built for every push...?

Reply:
im debugging 😢  can't get CI to work

Reply:
understood, we could set a filter to build for every push if the branch is of a certain format, like some the pattern "dev", "main", "petko_testing/***"